modules/db_parameter_group/variables.tf

```diff
@@ -7,7 +7,6 @@ variable "create" {
 variable "name" {
   description = "The name of the DB parameter group"
   type        = string
```

Comment:
There should be some default value for `name` because this module can be disabled conditionally `create = false` and thus the default value should be used.

Reply:
I added this part in https://github.com/terraform-aws-modules/terraform-aws-rds/pull/308/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR27 for this plus it also uses the identifier as a fallback which is more aligned with the previous default state (i.e. - if users did not provide anything then the option/parameter groups are created using the identifer value and they are prefixed). this also sort of makes sense from the submodule perspective because the name should be provided by users rather than a default - its just when its used from the root module that some value for the name must be provided

Reply:
yes we could add something - I wasn't sure though since option group doesn't contain a default https://github.com/bryantbiggs/terraform-aws-rds/blob/77c5e444c1aa5ef9798cb99cc40b4813891dbd5e/modules/db_option_group/variables.tf#L7 and users will still need to provide a value for `family`

I'll push up something now

Reply:
added in https://github.com/terraform-aws-modules/terraform-aws-rds/pull/308/commits/531ca748534aa7257797cc172d84a4c14dac4ff4